apis/supernova/src/handler/hypercube-handler.js

```diff
@@ -13,26 +12,51 @@ import removeMainMeasure from './utils/hypercube-helper/remove-main-measure';
 import removeAlternativeDimension from './utils/hypercube-helper/remove-alternative-dimension';
 import reinsertMainDimension from './utils/hypercube-helper/reinsert-main-dimension';
 import reinsertMainMeasure from './utils/hypercube-helper/reinsert-main-measure';
-// eslint-disable-next-line import/no-relative-packages
-import arrayUtil from '../../../conversion/src/array-util';
 
+/**
+ * HyperCubeHandler for managing hypercube data structure.
+ * @class HyperCubeHandler
+ * @description This class provides methods to handle hypercube properties, dimensions, and measures.
+ * @param {object} opts - Parameters to add a hypercube handlers
+ * @export
+ * @entry
+ * @example
+ * import HyperCubeHandler from '@nebula.js/stardust';
+ * const handler = new HyperCubeHandler({
+ *   app: qlikApp,
+ *   dimensionDefinition: { max: 2 },
+ *   measureDefinition: { max: 2 },
+ *   properties: {},
+ * });
+ *
+ * handler.setProperties({ qDimensions: [], qMeasures: [] });
+ * const dims = handler.getDimensions();
+ */
 class HyperCubeHandler extends DataPropertyHandler {
+  /**
+   * Creates an instance of HyperCubeHandler.
+   * @param {object} opts - Options for the handler.
+   */
```

Comment:
Remove this part, it will only add duplicates into the spec documentation, making it harder to read. (Same for DataPropertyHandler)